.github/workflows/automate.yaml

```
@@ -37,16 +37,19 @@ jobs:
           pip install -r requirements.txt
 
       - name: Run script
-        run: python main.py --days-needed 2 # Custom CLI Command
-#LINT.IF .env.example
+        run: python main.py --days-needed 2 PROMPTS_PATH MESSAGE_PATH
+        # Custom CLI Command
+
+        #LINT.IF .env.example
```

Nit: Resolve [check failure](https://github.com/acmcsufoss/acmOpportunities/actions/runs/5969957671/job/16196735903?pr=25#step:4:21).
```suggestion
#LINT.IF .env.example
```

---

Fix completed!